test/specs/core.specs.js

```diff
@@ -207,12 +207,15 @@ describe('riot-tmpl', function () {
 
     it('unwrapped keywords void, window and global, in addition to `this`', function () {
       data.$a = 5
-      expect(render('{' + (typeof window === 'object' ? 'window' : 'global') + '.globalVar }')).to.be(5)
+      expect(render('{ ' + (typeof window === 'object' ? 'window' : 'global') + ' }')).to.be.a('object')
+      expect(Object.keys(render('{ ' + (typeof window === 'object' ? 'window' : 'global') + ' }')).length).to.be(0)
+
+      expect(render('{' + (typeof window === 'object' ? 'window' : 'global') + '.globalVar }')).to.be(undefined)
       expect(render('{ this.$a }')).to.be(5)
       expect(render('{ void 0 }')).to.be(undefined)
       // without unprefixed global/window, default convertion to `new (D).Date()` throws here
-      data.Date = typeof window !== 'object' ? 'global' : 'window'
-      expect(render('{ new ' + data.Date + '.Date() }')).to.be.a('object')
+      data.Date = Date
```

Comment:
Comment on L216 is outdated now? If so we should remove it.

Reply:
It is outdated. Removed!